scripts/RVG02_B09_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# set base paths: customize as needed,
+# or comment out if you already set these up as global environment variables
+export NSDS_METADATA_PATH="${HOME}/Src/NSDSLab-NWB-metadata/"
+export NSDS_DATA_PATH='/clusterfs/NSDS_data/hackathon20201201/TTankBackup/'
+export NSDS_STIMULI_PATH='/clusterfs/NSDS_data/hackathon20201201/Stimuli/'
+
+SAVE_PATH="_test/"
+BLOCK_NAME="RVG02_B09"
+BLOCK_META="../tests/_data/RVG02/block_data.csv"
```

Comment:
In general, I think it would be better to have these files exist as part of a testing folder rather that as part of the repo. At least for this script, if `NSDS_DATA_PATH` exists (which it will need to for the script to work), you can set
`BLOCK_META=$NSDS_DATA_PATH/RVG02/block_data.csv`

IMO, this should be testing whether things are working based on the intended folder structure, not using the yamls in the repo. I do think it would be good to have metadata parsing tests, which could run on cvs/yaml files included in the repo.

Reply:
now pointing to a testing folder that stores metadata input drafts (although not `$NSDS_DATA_PATH`)